frontend/public/components/mounted-vol.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import {
+  getVolumeType,
+  getVolumeLocation,
+  getVolumeMountPermissions,
+  getVolumeMountsByPermissions,
+} from '../module/k8s/pods';
+import {
+  VolumeIcon,
+} from './utils';
+
+const Volume = ({volumeMounts}) => {
+  const kind = _.get(getVolumeType(volumeMounts.volume), 'id', '');
+  const loc = getVolumeLocation(volumeMounts.volume);
+  const name = volumeMounts.name;
+  const mountPermissions = getVolumeMountPermissions(volumeMounts);
+
+  return <React.Fragment>
+    <div className="row">
+      <div className="col-sm-2 col-xs-4">{name}</div>
+      {volumeMounts.mounts.map((m, i) => <React.Fragment key={i}>
+        <div className="col-sm-3 hidden-xs">{_.get(m, 'mountPath', '-')}</div>
+        <div className="col-sm-2 hidden-xs">{_.get(m, 'subPath', '-')}</div>
+      </React.Fragment>)}
+      <div className="col-sm-3 col-xs-4">
+        <VolumeIcon kind={kind} />
+        <span className="co-break-word">{loc && ` (${loc})`}</span>
+      </div>
+      <div className="col-sm-2 col-xs-4">{mountPermissions}</div>
+    </div>
+  </React.Fragment>;
+};
+
+export const MountedVolumes: React.SFC<MountedVolumesProps> = ({podTemplate}) => {
+
+  return <React.Fragment>
+    {_.isEmpty(getVolumeMountsByPermissions(podTemplate)) ?
+      <div className="cos-status-box">
+        <div className="text-center">No Volume Mounts Found</div>
+      </div>
+      :
+      <div className="co-m-table-grid co-m-table-grid--bordered">
+        <div className="row co-m-table-grid__head">
+          <div className="col-sm-2 col-xs-3">Name</div>
+          <div className="col-sm-3 col-xs-3">Mount Path</div>
+          <div className="col-sm-2 col-xs-3">SubPath</div>
+          <div className="col-sm-3 col-xs-3">Type</div>
+          <div className="col-sm-2 hidden-xs">Permissions</div>
```

Comment:
We need to add by the utilized by column. For pods, it should be a link to the container page. For other workloads, it should just be the container name (no a link).